java/integTest/src/test/java/glide/ConnectionTests.java

```diff
@@ -26,4 +39,173 @@ public void cluster_client() {
         var clusterClient = GlideClusterClient.createClient(commonClusterClientConfig().build()).get();
         clusterClient.close();
     }
+
+    @SneakyThrows
+    public GlideClusterClient createAzTestClient(String az) {
+        return GlideClusterClient.createClient(
+                        commonClusterClientConfig()
+                                .readFrom(ReadFrom.AZ_AFFINITY)
+                                .clientAZ(az)
+                                .requestTimeout(2000)
+                                .build())
+                .get();
```

Comment:
Indentation

Reply:
Is is managed by the java linter (spotless)